test/JDBC/input/functions/typeid-typename-dep-vu-prepare.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TYPE typeid_typename_dep_vu_prepare_t1 FROM INT;
+GO
+
+CREATE SCHEMA typeid_typename_dep_vu_prepare_s1;
+GO
+
+CREATE TYPE typeid_typename_dep_vu_prepare_s1.typeid_typename_dep_vu_prepare_t1 FROM INT;
```

Comment:
overall test cases should involve, 
1/ delimited identifiers along with multi byte and single byte chars with all kind of variations
2/ permission checks on type usage

Reply:
Added